### ISSUE TITLE
Nginx Docker build with GeoIP2 suddenly fails

### ISSUE THREAD

**Issue by Monokai** [nginxinc/docker-nginx#756]
After running fine for more than a year, suddenly I'm getting this error when trying to install the geoip2 package along with my Nginx Docker image:

`/bin/sh: ./abuild-module-geoip2/APKBUILD: line 27: syntax error: unexpected "fi"`

Looking back at the last successful deployment logs in my CI/CD, this is how it's supposed to be:

```
(…)
touch .deps-module-geoip2
===> Creating abuild-module-geoip2
+ . ./abuild-module-geoip2/APKBUILD
+ _base_version=1.23.3
+ _base_release=1
+ pkgname=nginx-module-geoip2
+ pkgver=1.23.3.3.4
+ pkgrel=1
+ pkgdesc='nginx 3rd-party GeoIP2 dynamic modules'
+ url=https://nginx.org/
+ arch=all
+ license='2-clause BSD-like license'
+ depends='nginx-r1.23.3 '
+ makedepends='linux-headers openssl-dev pcre2-dev zlib-dev libmaxminddb-dev'
+ provides='nginx-module-geoip2-r1.23.3 '
+ options='!check !strip'
+ pkgusers=
+ pkggroups=
+ install=nginx-module-geoip2.post-install
+ source='nginx-module-geoip2-1.23.3.tar.gz
	 ngx_http_geoip2_module-3.4.tar.gz
	COPYRIGHT
	'
+ builddir=/nginx-1.23.3
+ _modules_dir=usr/lib/nginx/modules
+ echo linux-headers openssl-dev pcre2-dev zlib-dev libmaxminddb-dev
+ apk add linux-headers openssl-dev pcre2-dev zlib-dev libmaxminddb-dev
(…)
```

And this is where I'm getting my error now:

```
(…)
touch .deps-module-geoip2
===> Creating abuild-module-geoip2
+ . ./abuild-module-geoip2/APKBUILD
+ _base_version=1.23.3
+ _base_release=1
+ pkgname=nginx-module-geoip2
+ pkgver=1.23.3.3.4
+ pkgrel=1
+ pkgdesc='nginx 3rd-party GeoIP2 dynamic modules'
+ url=https://nginx.org/
+ arch=all
+ license='2-clause BSD-like license'
+ depends='nginx~1.23.3 nginx-r1.23.3 '
+ makedepends='linux-headers openssl-dev pcre2-dev zlib-dev libmaxminddb-dev'
+ provides='nginx-module-geoip2-r1.23.3 '
+ options='!check !strip'
+ pkgusers=
+ pkggroups=
+ install=nginx-module-geoip2.post-install
/bin/sh: ./abuild-module-geoip2/APKBUILD: line 27: syntax error: unexpected "fi"
(…)
```

The last successful deployment was with Nginx version 1.23.3. It's the same version I'm trying now.

I'm not sure at which repository I have to create this issue. I've also created one here: https://github.com/leev/ngx_http_geoip2_module/issues/112 but have not received a reply yet.

**Comment by thresheek** [comment]
Hi @Monokai,

I don't think I can reproduce it with the usual `docker build --build-arg ENABLED_MODULES="geoip2" -t my-nginx-with-geoip2 -f Dockerfile.alpine .` in the current docker-nginx tree.

Any chance for you to peek into APKBUILD file internals, to see if it's mangled somehow?

**Comment by Monokai** [comment]
My bad, I'v fixed it.

I've had my own copy of an NGINX Dockerfile, including the commands to install the enabled modules. I still don't know why it suddenly didn't work anymore, but I just updated it with the latest code from the regular Dockerfile and it works again.

Looking at the differences, I saw that I still had some [code fixes from a previous issue](https://github.com/nginxinc/docker-nginx/issues/525#issuecomment-810179139) in there:

```sh
sed -i 's,subpackages="$pkgname-dbg",,' APKBUILD-module.in; \
sed -i 's,options="!check",options="!check !strip",' APKBUILD-module.in; \
```

I think this might have messed with the APKBUILD.

Anyway, fixed now!
